AsyncDisplayKit/ASDisplayNode+Subclasses.h

```diff
@@ -243,6 +243,31 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)visibilityDidChange:(BOOL)isVisible ASDISPLAYNODE_REQUIRES_SUPER;
 
 /**
+ * @abstract Called whenever the visiblity of the node changed.
+ *
+ * @discussion Subclasses may use this to monitor when they become visible.
```

Comment:
@lappp9 we should add that this is always called on the main thread.  Edit the comment for the prior version to say it is planned to be deprecated in a future release, and called in the same cases as the newer visibleStateDidChange: method.